servers/orbitmc/metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "orbitmc",
+    "name": "OrbitMc",
+    "description": "Place for ASIAN Pvpers",
+    "addresses": [
+        "mc.orbitmc.xyz"
+
+    ],
+    "primaryAddress": "mc.orbitmc.xyz",
+    "minecraftVersions": [
+        "1.7.*",
+        "1.8.*",
+
+    ],
+    "primaryMinecraftVersion": "1.8.9",
+    "primaryColor": "#FFD700",
+    "secondaryColor": "#FFFF00",
+    "primaryLanguage": "en",
+    "languages": [
+        "en"
+    ],
+    "primaryRegion": "AS",
+    "regions": [
+        "AS"   
+    ],
+    "gameTypes": [
+        "PvP",
+        "Bedwars"
+    ],
+    "website": "https://soon.soon.soon",
+    "store": "https://store.orbitmc.xyz",
+    "socials": {
+        "twitter": "None",

```

Comment:
if there is no twitter/telegram don't specify none just remove it from the JSON.

Reply:
okok